gtkmm.json

```diff
@@ -2,9 +2,18 @@
     "name": "gtkmm",
     "sources": [
         {
-            "sha256": "9beb71c3e90cfcfb790396b51e3f5e7169966751efd4f3ef9697114be3be6743",
+            "sha256": "30d5bfe404571ce566a8e938c8bac17576420eb508f1e257837da63f14ad44ce",
             "type": "archive",
-            "url": "https://ftp.gnome.org/pub/GNOME/sources/gtkmm/3.24/gtkmm-3.24.4.tar.xz"
+            "url": "https://ftp.gnome.org/pub/GNOME/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz"
+        },
+        {
+            "//": "Fix meson configure error Cannot find cairomm-1.16",
+            "type": "shell",
+            "commands": [
+                "sed -i 's/cairomm-1.0/cairomm-1.16/' meson.build",
+                "sed -i 's/pangomm-1.4/pangomm-2.48/' meson.build",
+                "sed -i 's/atkmm-1.6/atkmm-2.36/' meson.build"
+            ]
```

Comment:
no no and no. You are trying to build with incompatible version here. There is a reason why they have a different name. It because they are NO LONGER API compatible.

Reply:
Ok, I will resume this work in the weekend.